community/terra/qis_adv/quantum_walk.ipynb

```diff
@@ -0,0 +1,570 @@
+      "Requirement already satisfied: asn1crypto>=0.21.0 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cryptography>=1.3->requests-ntlm>=1.1.0->qiskit-terra<0.8,>=0.7->qiskit==0.7.1) (0.24.0)\n",
+      "Requirement already satisfied: pycparser in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cffi!=1.11.3,>=1.7->cryptography>=1.3->requests-ntlm>=1.1.0->qiskit-terra<0.8,>=0.7->qiskit==0.7.1) (2.19)\n",
+      "Requirement already satisfied: qiskit_aer==0.1.1 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (0.1.1)\n",
+      "Requirement already satisfied: numpy>=1.13 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from qiskit_aer==0.1.1) (1.16.2)\n",
+      "Requirement already satisfied: IBMQuantumExperience in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (2.0.4)\n",
+      "Requirement already satisfied: requests in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from IBMQuantumExperience) (2.21.0)\n",
+      "Requirement already satisfied: requests-ntlm in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from IBMQuantumExperience) (1.1.0)\n",
+      "Requirement already satisfied: urllib3<1.25,>=1.21.1 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests->IBMQuantumExperience) (1.24.1)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests->IBMQuantumExperience) (2018.11.29)\n",
+      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests->IBMQuantumExperience) (3.0.4)\n",
+      "Requirement already satisfied: idna<2.9,>=2.5 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests->IBMQuantumExperience) (2.8)\n",
+      "Requirement already satisfied: cryptography>=1.3 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests-ntlm->IBMQuantumExperience) (2.4.2)\n",
+      "Requirement already satisfied: ntlm-auth>=1.0.2 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from requests-ntlm->IBMQuantumExperience) (1.2.0)\n",
+      "Requirement already satisfied: cffi!=1.11.3,>=1.7 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (1.11.5)\n",
+      "Requirement already satisfied: six>=1.4.1 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (1.12.0)\n",
+      "Requirement already satisfied: asn1crypto>=0.21.0 in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (0.24.0)\n",
+      "Requirement already satisfied: pycparser in /Users/ryosuke/anaconda3/lib/python3.7/site-packages (from cffi!=1.11.3,>=1.7->cryptography>=1.3->requests-ntlm->IBMQuantumExperience) (2.19)\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
Please remove these install lines and add the Qiskit logo as per the other community tutorials.  `<img src="../../../images/qiskit-heading.png" alt="Note: In order for images to show up in this jupyter notebook you need to select File => Trusted Notebook" width="500 px" align="left">`